tests/src/test/scala/whisk/core/cli/test/WskWebActionsTests.scala

```diff
@@ -250,7 +172,7 @@ trait WskWebActionsTests
                 response.statusCode shouldBe 200
                 response.header("Access-Control-Allow-Origin") shouldBe "*"
                 response.header("Access-Control-Allow-Methods") shouldBe "OPTIONS, GET, DELETE, POST, PUT, HEAD, PATCH"
-                response.header("Access-Control-Allow-Headers") shouldBe "Authorization, Content-Type"
+                response.header("Access-Control-Allow-Headers") shouldBe "Authorization, Content-Type, User-Agent"
```

Comment:
@drcariel - requests to the controller can come from the cli, gui, apps using the rest interface, and web action invocations.  each can and will have a different user agent header, so the controller must be application agnostic wrt user agent header handling.  the controller does not enforce that a user agent header be part of an incoming request.  if the user agent header is included, then the controller will handle it like any other benign header the controller does not explicitly act upon.